nyc_data/ppe/models.py

```diff
@@ -166,6 +193,15 @@ def to_dataclass(self):
             source=self.source.display()
         )
 
+    def comparable_object(self):
+        
+        return { 
```

Comment:
I'd probably make the comparable object a frozenset, tuple, or namedtuple so that it was hashable